cola/widgets/commitmsg.py

```diff
@@ -638,4 +651,4 @@ def keyPressEvent(self, event):
     def setFont(self, font):
         SpellCheckTextEdit.setFont(self, font)
         fm = self.fontMetrics()
-        self.setMinimumSize(QtCore.QSize(1, fm.height() * 2))
+        self.setMinimumSize(QtCore.QSize(1, fm.height() * 2))
```

Comment:
This file now has "no newline at end of file".  Are you using sublime text?

https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842